src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapStoreScenario.scala

```diff
@@ -0,0 +1,50 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
+import com.linagora.gatling.imap.protocol.command.MessageRange._
+import com.linagora.gatling.imap.protocol.command.{MessageRanges, StoreFlags}
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import io.gatling.core.session.Expression
+
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+import scala.util.Random
+
+class ImapStoreScenario extends Simulation {
+  val numOfMailInInbox = 5000
+  val percentageOfMailToExpunge = 20.0
+  val silent = true
+
+  val userFeeder = csv("users.csv").circular
+  val userCount: Int = userFeeder.records.length
+
+  val populateMailbox = exec(imap("append").append("INBOX", Option.empty[Seq[String]], Option.empty[Calendar],
+    """From: expeditor@example.com
+      |To: recipient@example.com
+      |Subject: test subject
+      |
+      |Test content
+      |abcdefghijklmnopqrstuvwxyz
+      |0123456789""".stripMargin).check(ok))
+
+  val populateInbox = repeat(numOfMailInInbox)(pause(5 milliseconds).exec(populateMailbox))
```

Comment:
I put them because otherwise I get issue with cyrus maybe it could not handle to much // append request. The aim wasn't to test append but expunge !